### PR TITLE
Regenerate package-lock.json

### PR DESCRIPTION
**Pull request by XhmikosR**
This brings package.json and package-lock.json versions up to date.

Do note that due to the way dependabot works, regenerating the lock file is needed after some time, otherwise the deps tree isn't flattened as much as it could be.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Not sure how to proceed with the `jest/valid-describe`. We'll hit this later anyway. Should I globally disable the rule?

**Comment by hudochenkov** [comment]
Disable it with disable comments in a file.

**Comment by XhmikosR** [comment]
Might make more sense to first merge #4504

**Comment by ybiquitous** [comment]
@XhmikosR will you rebase #4504?

**Comment by XhmikosR** [comment]
Sure thing, but let's run dependabot one more time before that.

File: package.json [hunk 1]
```diff
@@ -185,15 +185,24 @@
...
-      "node/no-unsupported-features/es-builtins": ["error", {
-        "version": ">=10.18.0"
-      }],
-      "node/no-unsupported-features/es-syntax": ["error", {
-        "version": ">=10.18.0"
-      }],
-      "node/no-unsupported-features/node-builtins": ["error", {
-        "version": ">=10.18.0"
-      }]
+      "node/no-unsupported-features/es-builtins": [
```
Comment (XhmikosR) [inline package.json]: @hudochenkov these changes are done automatically by npm

**Pull request merged**